package.json

```diff
@@ -24,23 +24,38 @@
   },
   "homepage": "https://github.com/odoo/jabberwock#readme",
   "devDependencies": {
-    "@types/jest": "^24.0.18",
+    "@types/chai": "^4.2.1",
+    "@types/dom-inputevent": "^1.0.5",
+    "@types/mocha": "^5.2.7",
     "@types/node": "^12.7.2",
+    "@types/sinon": "^7.0.13",
     "@typescript-eslint/eslint-plugin": "^2.0.0",
     "@typescript-eslint/parser": "^2.0.0",
     "acorn": "^7.0.0",
+    "chai": "^4.2.0",
+    "concurrently": "^4.1.2",
     "eslint": "^6.2.2",
     "eslint-config-prettier": "^6.1.0",
     "eslint-plugin-prettier": "^3.1.0",
     "finalhandler": "^1.1.2",
-    "jest": "^23.6.0",
+    "karma": "^4.3.0",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-firefox-launcher": "^1.2.0",
+    "karma-mocha": "^1.3.0",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-sinon": "^1.0.5",
+    "karma-webpack": "^4.0.2",
+    "lodash": "^4.17.15",
```

Comment:
Why ?

Reply:
I forgot to remove it, it should not be there